sagemaker-python-sdk/mxnet_gluon_mnist/mnist.py

```diff
@@ -103,13 +105,11 @@ def train(current_host, channel_input_dirs, hyperparameters, hosts, num_gpus):
 
 def save(net, model_dir):
     # save the model
-    y = net(mx.sym.var('data'))
-    y.save('%s/model.json' % model_dir)
-    net.collect_params().save('%s/model.params' % model_dir)
+    y = net.export('%s/model'% model_dir)
```

Comment:
do we need to set `y` anymore or can we remove it here?

Reply:
good point